src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,59 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_SIZE = 10;
+    private static final int DEFAULT_ARRAY_INDEX = -1;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default for numbers